src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -319,6 +319,42 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # Default 0-sized value means unlimited rate.
 #tx_rate_limiter_max_rate = 0
 
+# pvpanic device is a simulated ISA device, through which a guest panic
+# event is sent to qemu, and a QMP GUEST_PANICKED event is generated.
+# This allows management apps to be notified and respond to the event.
+#
+# Note:
+#   To enable pvpanic device, you must set
+#   CONFIG_PVPANIC=y for your kernel.
+#pvpanic_enabled=false
+
+# If true, when GUEST_PANICKED event occurred,
+# guest memeory will be dumped using
+# "dump-guest-memory" command.
```

Comment:
I'm not sure we need the `using "dump-guest-memory" command` comment since:

- That might change in the future?
- It's a QMP command which most users won't know anything about.

If we do keep the comment, I think we need to explain this is a Qemu QMP command.